recipes/anypytools/bld.bat

```diff
@@ -0,0 +1,6 @@
+"%PYTHON%" setup.py install 
+
+cd "%PREFIX%"
+mkdir AnyPyToolsTutorial
```

Comment:
Should we maybe put this somewhere else in the `%PREFIX%`? Not really sure where. Maybe `share`? @ocefpaf @pelson, thoughts on this one?